openupgrade_scripts/scripts/hr_contract/15.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,6 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.convert_field_to_html(env.cr, "hr_contract", "notes", "notes")
```

Comment:
Please copy the column on pre to preserve the original content, and do this on post.